scala3doc/dotty-docs/docs/blog/index.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: main
```

Comment:
Can we delete `docs/blog/index.html`?

Reply:
Same as here: https://github.com/lampepfl/dotty/pull/10081#discussion_r512655872